js/main.js

```diff
@@ -1,2 +1,70 @@
 /* global data */
 /* exported data */
+
+var $entryForm = document.querySelector('[data-view="entry-form"]');
+var $entryUrl = document.querySelector('#entry-url');
+var $entryTitle = document.querySelector('#entry-title');
+var $entryNotes = document.querySelector('#entry-notes');
+var $entryImage = document.querySelector('.entry-image > img');
+var photoUrl = '';
+var photoTitle = '';
+var notes = '';
+
+if (localStorage.getItem('data')) {
+  for (var key in data) {
+    var previousDataJSON = localStorage.getItem('data');
+    var previousData = JSON.parse(previousDataJSON);
+    data[key] = previousData[key];
+  }
+}
```

Comment:
All localStorage related code should be in `data.js`, where the variable it is storing is located.

Reply:
alright, I'll get to it